projects/plugins/jetpack/extensions/blocks/videopress/v6/edit.js

```diff
@@ -39,16 +39,36 @@ export default function VideoPressEdit( { attributes, setAttributes } ) {
 		controls,
 	} );
 
-	// Uploading file to backend states.
-	const [ uploadingProgress, setUploadingProgress ] = useState( [] );
+	/*
+	 * Tracking state when uploading the video file.
+	 * uploadingProgress is an array with two items:
+	 *  - the first item is the upload progress
+	 *  - the second item is total
+	 */
+	const [ uploadingProgress, setUploadingProgressState ] = useState( [] );
+
+	/*
+	 * Uploading progress handler, wrapper by the useCallback() hook
+	 * to avoid re-rendering the component when the progress is updated.
+	 */
```

Comment:
Don't get the point here. We are still updating the state and generating a new render.